hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerTask.java

```diff
@@ -361,23 +355,20 @@ public List<ContainerBalancerTaskIterationStatusInfo> getCurrentIterationsStatis
         findTargetStrategy.getSizeEnteringNodes()
             .entrySet()
             .stream()
-            .filter(Objects::nonNull)
             .filter(datanodeDetailsLongEntry -> datanodeDetailsLongEntry.getValue() > 0)
-            .collect(Collectors.toMap(
-                    entry -> entry.getKey().getUuid(),
-                    entry -> entry.getValue() / OzoneConsts.GB
-                )
+            .collect(
+                ConcurrentHashMap::new,
```

Comment:
We can use Map itself only here, no code change is required, if base map is threadsafe. Here new map no need be threadsafe.